st2common/st2common/service_setup.py

```diff
@@ -186,6 +187,9 @@ def setup(service, config, setup_db=True, register_mq_exchanges=True,
         register_service_in_service_registry(service=service, capabilities=capabilities,
                                              start_heart=True)
 
+    if sys.version_info[0] == 2:
+        LOG.warning(PYTHON2_DEPRECATION)
```

Comment:
Looking at the code, maybe it makes more sense to move this warning at earlier point, just after the
```
    LOG.debug('Using Python: %s (%s)' % (version, sys.executable))
```
?

Reply:
If you do that then it doesn't go to the log files. As it is before the logger is setup (I originally put it there until I realised that!)

Reply:
Cool! So I've made same mistake :) 
Thanks for the explanation.